backend/routing_service.py

```
@@ -0,0 +1,177 @@
+        2. Match against department-specific rules
+        3. Fall back to default jurisdiction level
+
+        Args:
+            grievance_data: Dictionary containing grievance details
+            db: Database session
+
+        Returns:
+            Jurisdiction object or None if no match found
+        """
+        category = grievance_data.get('category')
+        pincode = grievance_data.get('pincode')
```

Ruff (0.14.14)

[error] Local variable `pincode` is assigned to but never used

Remove assignment to unused variable `pincode`

(F841)